api-controllers/FabrikBaseController.js

```diff
@@ -205,6 +206,16 @@ class FabrikBaseController extends BaseController {
   getPlan(plan_id) {
     return catalog.getPlan(plan_id);
   }
+
+  static getDashboardUrl(serviceId, planId, instanceId) {
```

Comment:
Why is it moved here? Dashboard is only used from `ServiceBrokerApiController.js` so wouldn't it make sense to have it there? For other ApiControllers, it does not make any sense.